src/test/java/org/jenkinsci/plugins/plaincredentials/BaseTest.java

```diff
@@ -56,7 +56,7 @@ public void secretTextBaseTest() throws IOException {
     public void secretFileBaseTest() throws IOException, URISyntaxException {
         DiskFileItem fileItem = createEmptyFileItem();
         
-        FileCredentialsImpl credential = new FileCredentialsImpl(CredentialsScope.GLOBAL, CRED_ID, "Test Secret file", fileItem, "keys.txt", Base64.encode(fileItem.get()).toString());
+        FileCredentialsImpl credential = new FileCredentialsImpl(CredentialsScope.GLOBAL, CRED_ID, "Test Secret file", fileItem, "keys.txt", SecretBytes.fromBytes(fileItem.get()));
```

Comment:
Actually we should have at least one test with the deprecated constructor to ensure its code works